lib/jazzy/sourcekitten.rb

```diff
@@ -88,19 +132,16 @@ def self.documented_child?(doc)
     end
 
     def self.should_document?(doc)
-      unless doc['key.usr']
-        warn "`#{doc['key.name']}` has no USR. First make sure all modules "  \
-              'used in your project have been imported. If all used modules ' \
-              'are imported, please report this problem by filing an issue '  \
-              'at https://github.com/realm/jazzy/issues along with your '     \
-              'Xcode project. If this token is declared in an `#if` block, '  \
-              'please ignore this message.'
-        return false
-      end
       return false if doc['key.doc.comment'].to_s.include?(':nodoc:')
 
-      # Always document extensions, since we can't tell what ACL they are
-      return true if doc['key.kind'] == 'source.lang.swift.decl.extension'
+      # Document extensions & enum elements, since we can't tell their ACL.
+      type = SourceDeclaration::Type.new(doc['key.kind'])
+      return true if type.enum_element?
+      if type.extension?
+        return (doc['key.substructure'] || []).any? do |subdoc|
```

Comment:
Except it's less safe, since `Array()` coerces anything to an array, including arrays, nil, and a single object.